service/plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/exception/PluginClientException.java

```diff
@@ -0,0 +1,19 @@
+package com.solace.maas.ep.event.management.agent.plugin.exception;
```

Comment:
This is a new Generic Plugin Exception that Camel will look for to know when a scan has failed.